src/components/uploadpost/UploadPost.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  StyleFormConatiner,
+  StyleInput,
+  CancelButton,
+  DoneButton,
+  LargeInput,
+  StyledLabel,
+  InputRow
+} from './StyledUpload.js';
+import { supabase } from '../../../supabase/supabase.js'; // supabase 사용하려면
+import SelectCategory from './../SelectCategory/SelectCategory';
+import { useDispatch, useSelector } from 'react-redux';
+
+const UploadPost = () => {
+  const [title, setTitle] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [passwords, setPassword] = useState('');
+  const [content, setContent] = useState('');
+  const [time, setTime] = useState('');
+  const [participants, setParticipants] = useState('');
+  const [coverImage, setCoverImage] = useState('');
+  const [imageUrl, setImageUrl] = useState('');
+
+  const category = useSelector((state) => state.UploadPostSlice.category);
+  const lng = useSelector((state) => state.mapSlice.lng);
+  const lat = useSelector((state) => state.mapSlice.lat);
```

Comment:
리덕스라 무시하셔도 됩니다!

```suggestion
  const {category, lng, lat} = useSelector((state) => {
    return {
       category: state.UploadPostSlice.category,
       lng: state.mapSlice.lng,
       lat: state.mapSlice.lat
    }
  }, shallowEqual);
```
로 작성하시면 한번에도 작성 가능합니다!